src/reactapp/src/components/shippingMethod/components/ShippingMethodList.jsx

```diff
@@ -44,6 +45,18 @@ function ShippingMethodList() {
         {_objToArray(methodList).map((method) => {
           const { id: methodId, carrierTitle, methodTitle, price } = method;
           const methodName = `${carrierTitle} (${methodTitle}): `;
+          const MethodRenderer = methodRenderers[methodId];
+
+          if (MethodRenderer) {
+            return (
+              <MethodRenderer
```

Comment:
So you want the `<li>` element to be defined in the method renderer and not here?

Reply:
I think you have a valid point. It is better we keep <li> here in this component.